README.md

```diff
@@ -68,7 +68,7 @@ googleAnalytics = "" # Enable Google Analytics by entering your tracking id
   toc = true # Enable Table of Contents
   post_navigation = true # Show post navigation at bottom of pages if true
   post_meta = ["date", "categories", "translations"] # Order of post meta information
-  #mainSections = ["post", blog", "news"] # specify section pages to show on home page and the "Recent articles" widget
+  #mainSections = ["posts", "blog", "news"] # specify section pages to show on home page and the "Recent articles" widget
```

Comment:
> Add " for mainSections

Nice catch! 👌 @jfmatth

> ...and match to Hugo quickstart section

In our [exampleSite content](https://github.com/Vimux/Mainroad/tree/master/exampleSite/content) we use `post` section, not `posts`, so it seems illogical to use different sections for demo content and README. 

[Hugo Quick Start guide](https://gohugo.io/getting-started/quick-start/) is an example of how to create your first Hugo site, nothing more, nothing less. Furthermore, it uses a different theme.

Could you please update your PR?

```suggestion
  #mainSections = ["post", "blog", "news"] # specify section pages to show on home page and the "Recent articles" widget
```